src/app/tests/suites/certification/Test_TC_SMOKECO_1_1.yaml

```diff
@@ -23,23 +23,23 @@ config:
     endpoint: 1
 
 tests:
-    - label: "Commission DUT to TH"
+    - label: "Step 1: Commission DUT to TH"
```

Comment:
a lot of these changes are not "add method to get eventNumber". Please split them in separate PRs so that we can review them individually.